db/migrate/20171212084905_change_game_mode_and_player_types_null.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeGameModeAndPlayerTypesNull < ActiveRecord::Migration[5.1]
+  def change
+    change_column :events, :player_type, :integer, :null => true
```

Comment:
I'm still a bit surprised that this isn't important for you guys at all. Single players can only join teams with the player_type :single and teams will only be able to join events with the player_type :team. So if you let this column be 0 the join and leave methods will not work for you anymore. If you want to add both you can just add that option to the enum and change the code in join and leave to permit both if the option is selected.

Otherwise noone will be able to join a rankinglist (maybe this is intended?)